src/taglibs/migrate/all-tags/legacy-nested-tag.js

```diff
@@ -0,0 +1,16 @@
+const legacyNestedTagSyntax = /^.*:(.*)$/;
```

Comment:
I doubt this is a problem, but this will technically match a tag in this format: `<:helloworld>`